app/models/post.rb

```diff
@@ -1,5 +1,19 @@
 class Post < ApplicationRecord
-  belongs_to :user
+  belongs_to :author, class_name: 'User', foreign_key: :author_id
   has_many :comments
   has_many :likes
+
+  def update_post_counter
```

Comment:
- [ ] **[OPTIONAL]** We can also add a callback for this method so that whenever a post is created, the method is called on its own.

  ```
  after_save :update_post_counter
  ```